tests/test_subquery_is_synced.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import pytest
+from typing import List
+from scripts.utils.work_with_data import get_network_list
+
+payload = "{\"query\":\"query{\\n  _metadata{\\n    chain\\n    lastProcessedHeight\\n    targetHeight\\n  }\\n}\",\"variables\":{}}"
+headers = {
+    'Host': 'gateway.subquery.network',
+    'Connection': 'Keep-Alive',
+    'User-Agent': 'okhttp/4.11.0',
+    'Content-Type': 'application/json; charset=UTF-8'
+}
+
+
+def get_unique_subquery_urls(network_list: List[dict]) -> List[str]:
+    subquery_urls = set()
+    for chain in network_list:
+        external_api = chain.get('externalApi', {})
+        for api_type, api_list in external_api.items():
+            if isinstance(api_list, list):
+                for api in api_list:
+                    if api.get('type') == 'subquery':
+                        subquery_urls.add(api['url'])
+            elif isinstance(api_list, dict) and api_list.get('type') == 'subquery':
+                subquery_urls.add(api_list['url'])
+    return list(subquery_urls)
+
+
+@pytest.fixture(scope="module")
+def subquery_projects():
+    network_file_path = "chains/v20/chains.json"
```

Comment:
Lets use env variable here:
```suggestion
    network_file_path = os.getenv('CHAINS_JSON_PATH', "chains/v20/chains.json")
```